netbench/netbench-driver/src/bin/netbench-driver-s2n-quic-client.rs

```diff
@@ -20,8 +20,8 @@ pub struct Client {
     #[structopt(flatten)]
     opts: netbench_driver::Client,
 
-    #[structopt(long)]
-    disable_gso: bool,
+    #[structopt(long, env = "DISABLE_GSO")]
+    disable_gso: bool
```

Comment:
Think you need this to fix the **`netbench / rustfmt (pull_request)`** CI

```suggestion
    disable_gso: bool,
```

Reply:
Thanks for the catch!